docs/misc/projects.rst

```diff
@@ -161,3 +161,15 @@ Suite of RL environments focused on using a simulated tactile sensor as the prim
 | GitHub: https://github.com/ac-93/tactile_gym
 | Paper: https://arxiv.org/abs/2106.08796
 | Website: `tactile-gym website <https://sites.google.com/my.bristol.ac.uk/tactile-gym-sim2real/home>`_
+
+rl-exploration-baselines
+------------------------
```

Comment:
Would you agree with the following change?

```suggestion 
RLeXplore
---------
```

Reply:
Of course. Hope we can make more progress on practical intrinsic reward modules together!

Reply:
It could be great! What about the proposed changed https://github.com/DLR-RM/stable-baselines3/pull/1246#discussion_r1058270352?

Reply:
The change is okay for me. Thank you!